easybuild/easyconfigs/a/ASE/ASE-3.10.0-intel-2016.02-GCC-4.9-Python-2.7.11.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'ASE'
+version = '3.10.0'
+
+homepage = 'https://wiki.fysik.dtu.dk/ase/'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+
+source_urls = ['https://pypi.python.org/packages/source/a/ase/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+
+pyver = '2.7.11'
+pyshortver = '.'.join(pyver.split('.')[:2])
+versionsuffix = '-Python-%s' % pyver
```

Comment:
we have a better way of handling this now via the `%(pyver)s` template, can you please update this easyconfig accordingly?

```python
name = 'ASE'
version = '3.10.0'
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [
    ('Python', '2.7.11'),
]
...
    'dirs': ['lib/python%(pyshortver)s/site-packages/%(namelower)s'],
```

then you can drop the `pyver = ` and `pyshortver = ` lines entirely